app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -28,6 +28,32 @@
       </div>
     </div>
 
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-1 govuk-font-colour-white">
+          <%= t("coronavirus_landing_page.stay_at_home.pretext") %>
+        </p>
+        <ul class="govuk-list govuk-list--bullet govuk-font-colour-white">
+          <% t("coronavirus_landing_page.stay_at_home.list").each do | item | %>
+            <li><%= item %></li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+
+    <div class="govuk-grid-row covid__page-header-guidance-section">
+      <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body govuk-font-colour-white govuk-!-margin-bottom-1">
+          <%= t("coronavirus_landing_page.guidance.pretext") %>
+        </p>
+        <p class="govuk-body govuk-font-colour-white">
```

Comment:
`govuk-font-colour-white` isn't needed on this para, as the only content is a link that already has a white font colour applied.